.github/workflows/detect_version_bump.yml

```diff
@@ -0,0 +1,30 @@
+name: Version Bump merged
+
+on:
+  pull_request:
+    types:
+      - "closed"
+    branches:
+      - "main"
+
+jobs:
+  trigger-release:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged &&
+        startsWith(github.head_ref, 'bump_version_to_') &&
+        startsWith(github.event.pull_request.title, 'Bump version') &&
+        contains(github.event.pull_request.title, ' → ')
+    environment: Create Release
+    steps:
+      - name: Get new version
+        id: get-new-version
+        run: |
+          NEW_VERSION=$(echo ${{ github.head_ref }} | cut -d _ -f4 )
+          echo "::set-output name=version::$NEW_VERSION"
+      - name: Create Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        with:
+          tag_name: v${{ steps.get-new-version.outputs.version }}
+          release_name: v${{ steps.get-new-version.outputs.version }} Release
```

Comment:
`bump_version.sh` supports tagging a version as both a pre-release & as a dev release. Do you think we'd want to parse out that information and use it to set the `prerelease` and `draft` inputs, respectively, on the `create-release` action?

Reply:
That's an interesting idea. I was mostly focused on the normal release process, but pre-release seems like an important aspect to also include for handling the case of an alpha release.

Reply:
Created #74